src/hypothesis/internal/debug.py

```diff
@@ -42,3 +45,28 @@ def minimal(
         settings=settings,
         random=random,
     )
+
+
+HYPOTHESIS_ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Wait, no, that's not right. Your way drops the ..

I'll tinker with it to get it tidier though.